ci/build-check.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
```

Comment:
Maybe, but it's not very relevant in this case since this code only executes on Fedora-derived containers and in fact directly references `yum`.